src/test/java/org/jabref/model/entry/AuthorListTest.java

```diff
@@ -1152,4 +1273,70 @@ public void correctNamesWithOneComma() throws Exception {
         expected = new Author("José María", "J. M.", null, "Rodriguez Fernandez", null);
         assertEquals(new AuthorList(expected), AuthorList.parse("Rodriguez Fernandez, José María"));
     }
+
+    @Test
+    public void equalsFalseDifferentOrder() {
```

Comment:
Usually you don't need to write tests for equals or hashCode. These methods are usually automatically generated by the IDE anyways. (You can keep the tests here, was just a general remark for next time).

Reply:
Ok! General remarks are needed and appreciated! 😄